src/app/cube/content-pages/contribute-page/copy.ts

```diff
@@ -0,0 +1,162 @@
+export const sections = {
+    contribute: {
+        title: 'Contribute',
+        topics: [
+            {
+                title: 'Starting Out',
+                questions: [
+                    {
+                        question: 'Your Profile',
+                        answer: 'Your profile is a simple webpage that shows your basic public user information on CLARK. The ' +
+                         'information includes your chosen name, associated institution, email, biography, and all of your created ' +
+                         'learning objects. You will also see your unsubmitted learning objects from your profile, but only your ' +
+                         'released learning objects are viewable by the public. You can edit your profile information at any time ' +
+                         'with the <em>Edit Profile</em> button.'
+                    },
+                    {
+                        question: 'Your Dashboard',
+                        answer: 'Your dashboard is where you will access and create all of your learning objects. The dashboard is the ' +
+                        'primary way to begin designing a learning object. All draft (non-released) and released learning objects can be' +
+                        ' viewed within the dashboard. In addition, a status designates all learning objects while on the dashboard.'
```

Comment:
The second sentence here doesn't hold much meaning. What does a status designate for a learning object? Why are they there?